### DISCUSSION TITLE
Non-english (polish) spell checking not working

### DISCUSSION THREAD

**Discussion by isorokuYam** [vkbo/novelWriter#1398] [Help] [answered]
Hello,
I'm trying to use spell check with polish dictionary, but after setup all text is underlined red and no suggestions,
I've done everything as described in manual I belive. dic and aff files into local data and selected language in preferences.
![image](https://user-images.githubusercontent.com/52502292/226463895-8ad2eb12-2621-4c41-98a1-6b7b7bd905a0.png)
![image](https://user-images.githubusercontent.com/52502292/226463993-81c20d64-c50b-4739-98f6-c2df934a85b6.png)
![image](https://user-images.githubusercontent.com/52502292/226464170-8c001b45-fb50-43d9-8e74-4b84d80e824d.png)
![image](https://user-images.githubusercontent.com/52502292/226464305-39a795a7-7c9c-40e6-9723-32163d7319a7.png)

**Comment by vkbo** [comment, marked as answer]
First of all, please use the discussion section for help request, not the issue tracker.

There are sometimes some tricky issues with downloading files from https://cgit.freedesktop.org/libreoffice/dictionaries/tree/pl_PL, so if you just clicked on the files, that may be the issue. If you instead download the files by right-clicking on "plain" and saving them, that should work. If you just click on the file name and download it, you actually get a HTML file, not the .aff or .dic file.

The Enchant spell check package is designed for Linux and MacOS, and doesn't integrate with Windows, so you have to do this manual workaround. I plan to add a tool to automate this to avoid such issues, but I haven't gotten to it yet.

**Reply by vkbo** [reply]
I've updated the documentation and added a note about this. It's not at all obvious that the links are not the actual files if you don't notice it's a repository and not a file tree.

https://novelwriter.io/docs/int_customise.html

**Reply by isorokuYam** [reply]
Hi, 
thank you for answer and moving this to proper section.

I was downloading by right-click on file name, which indeed was downloading html, but with .dic/.aff extension (atleast in Edge), which was misleading. Problem solved, when downloaded from "plain" link. Thanks!

**Comment by lucastucious** [comment]
+1 To this. I was struggling to find why spell check wasn't working. You should specify in docs that we should download the "plain" file, and note the file on the repo

**Reply by vkbo** [reply]
It's already in the docs: https://novelwriter.io/docs/int_customise.html#spell-check-dictionaries
